README.md

```diff
@@ -32,18 +38,67 @@ include:
 
 ## Installation
 
-For releases from August 2022 it is recommended that you [create and use a Github PAT](https://docs.github.com/en/github/authenticating-to-github/keeping-your-account-and-data-secure/creating-a-personal-access-token) to install the latest version of this package. Once you have the PAT, run the following:
+From July 2023 `insightsengineering` packages are available on [r-universe](https://r-universe.dev/).
+
+```r
+# stable versions
+install.packages('teal.code', repos = c('https://insightsengineering.r-universe.dev', 'https://cloud.r-project.org'))
+
+# install.packages("pak")
+pak::pak("insightsengineering/teal.code@*release")
+```
+
+Alternatively, you might also use the development version.
+
+```r
+# beta versions
+install.packages('teal.code', repos = c('https://pharmaverse.r-universe.dev', 'https://cloud.r-project.org'))
+
+# install.packages("pak")
+pak::pak("insightsengineering/teal.code")
+```
+
+## Usage
+
+To understand how to use this package, please refer to the [Getting Started](https://insightsengineering.github.io/teal.code/latest-tag/articles/teal-code.html) article, which provides multiple examples of code implementation.
+
+Below is the showcase of the example usage
+
+```r
+library(teal.code)
+my_qenv <- new_qenv(env = list2env(list(x = 5)), code = "x <- 5")
+my_qenv
+Parent: <environment: package:teal.code>
+Bindings:
+• x: <dbl> [L]
+```
+
+```r
+qenv_2 <- eval_code(my_qenv, "y <- x * 2") |> eval_code("z <- y * 2")
+qenv_2
+<environment: 0x00000135b544cfe8> [L]
+Parent: <environment: package:teal.code>
+Bindings:
+• x: <dbl> [L]
+• y: <dbl> [L]
+• z: <dbl> [L]
+```
+
+```r
+qenv_2[["y"]]
+[1] 10
+```
 
 ```r
-Sys.setenv(GITHUB_PAT = "your_access_token_here")
-if (!require("remotes")) install.packages("remotes")
-remotes::install_github("insightsengineering/teal.code@*release")
+cat(paste(get_code(qenv_2), collapse = "\n"))
+x <- 5
+y <- x * 2
+z <- y * 2
 ```
```

Comment:
would it be possible to somehow distinguish command from the output? Right not it's not super readable:
<img width="320" alt="image" src="https://github.com/insightsengineering/teal.code/assets/12943682/a2bd14e7-c569-410f-b527-cad9f3c6647d">

e.g. `dplyr` does `#>` upfront any of the line in the output
there might be other ways as well

(please pass this on to the other readme updates PRs if exist)